.github/workflows/pull-request.yml

```
@@ -37,6 +37,28 @@ jobs:
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8 black pydocstyle
+
+      - name: Run Black Formatter Check
+        run: |
+          black --check .      
+            
+      - name: Run Flake8 Linter
+        run: |
+          flake8 . --ignore E402,E722
+        
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)